industrial_ci/ci_main.sh

```diff
@@ -325,4 +325,4 @@ cd $TARGET_REPO_PATH  # cd back to the repository's home directory with travis
 pwd
 
 HIT_ENDOFSCRIPT=true
-success 0
```

Comment:
Thanks for the headsup. If there any necessary change let us know further via the comments or PRs. IMO utility functions need to be called within the current process rather than in sub process, so `source`-ing is still the best way to use `util.sh`.